Sources_v3/Workers/UserUpdater.swift

```diff
@@ -64,6 +64,38 @@ class UserUpdater<ExtraData: ExtraDataTypes>: Worker {
                 }
             }
     }
+    
+    /// Flags or unflags the user with the provided `userId` depending on `flag` value.
+    /// - Parameters:
+    ///   - flag: The indicator saying whether the user should be flagged or unflagged.
+    ///   - userId: The user identifier.
```

Comment:
_what_ user identifier? The one who flags or the one who is flagged?

Reply:
fixed in 63b93cd2dacb4ad249e2ec74858fc46725feec3d